docs/tutorials/01-fee/08-packet-filter.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Hermes Packet Filtering by Fee
+sidebar_label: Hermes Packet Filtering by Fee (Optional)
+sidebar_position: 8
+slug: /fee/packet-filter
+---
+
+# Hermes Packet Filtering by Fee
+
+Hermes provides a way for relayers to only relay packets that have a fee greater than a certain amount.
+In this section, we will configure this option to only relay packets that have a fee greater than 30 `token`.
+Currently, Hermes only supports filtering by the receive packet fee.
+Filtering by acknowledgement packet fee or timeout packet fee is not supported.
+
+## Configure Hermes
+
+To configure Hermes to filter packets by fee, we need to add the following to the `config.toml` file:
+
+```toml
+[chains.packet_filter]
+policy = 'allow'
+list = [
+  ['ica*', '*'],
+  ['transfer', '*'],
+]
+
+[chains.packet_filter.min_fees.'*']
+recv = [ { amount = 30, denom = 'token' } ]
+```
+
+Here is a full example of the `config.toml` file:
+
+```toml reference title="hermes/filtered_config.toml"
+https://github.com/srdtrk/cosmoverse2023-ibc-fee-demo/blob/1ddac03efdf6d403126c3f5ad067fd708e2e410a/hermes/filtered_config.toml
+```
+
+You can clone this using the following command:
```

Comment:
```suggestion
You can copy this using the following command:
```

usage of `clone` here feels a bit weird.

Reply:
done